### PR TITLE
feat: paparazzi tests and reports

### LINKED ISSUE #7: Setup the CI to run screenshot testing
We just want it to run like it did on the LBC app

### PR DESCRIPTION
**Pull request by SimonMarquis**
Closes #7

### CODE REVIEW

File: .github/actions/archive-paparazzi-reports/action.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      uses: actions/upload-artifact@v3
+      with:
+        name: paparazzi-html
+        path: '**/build/reports/paparazzi'
+    - name: 'Archive deltas'
+      if: ${{ hashFiles('**/out/failures/**.png') != '' }}
+      uses: actions/upload-artifact@v3
+      with:
+        name: paparazzi-delta
+        path: '**/out/failures/**.png'
```
Comment (soulcramer) [inline .github/actions/archive-paparazzi-reports/action.yml:16]: SHouldn't this be '**/out/failures/delta-**.png' or do we want to upload all screenshots?

Comment (SimonMarquis) [reply .github/actions/archive-paparazzi-reports/action.yml:16]: I suppose we want both. `Delta-*` are composited images that could differ from the real content.

Comment (soulcramer) [reply .github/actions/archive-paparazzi-reports/action.yml:16]: Actually they have the previous one on the left side | the delta in the middle | the new one on the right side so imo we can just use these

Comment (SimonMarquis) [reply .github/actions/archive-paparazzi-reports/action.yml:16]: Yes I know they combine both images, but the resulting composition does not correspond exactly to what as been produced by paparazzi. So I prefer we keep both files for now.

Comment (soulcramer) [reply .github/actions/archive-paparazzi-reports/action.yml:16]: Good to me

**Comment by SimonMarquis** [comment]
I think this looks good enough for now, cc @soulcramer

**Review by soulcramer — APPROVED** [review]
LGTM :awesome:

**Pull request merged**